docs/tutorial/4-deploy-temporal.md

```diff
@@ -0,0 +1,179 @@
+# Deploy Temporal
+
+MAAS Site Manager uses Temporal Workflows for long-running tasks. Follow the steps below to deploy charmed Temporal.
+
+
+## Deploy Temporal Server
+
+First, add another Juju model for Temporal, deploy the Temporal server k8s charm, and wait for it to enter a `blocked/idle` state:
+
+```bash
+juju add-model temporal
+juju deploy temporal-k8s --config num-history-shards=4
+juju status --watch 5s  # wait for blocked/idle
+```
+
+Next, switch back to the `msm` model and create an offer for `postgresql-k8s`:
+
+```bash
+juju switch msm
+juju offer postgresql-k8s:database pgsql
```

Comment:
move the creation of this offer to section 2 (pgsql deployment), to avoid making the user going back and forth between models.